README.md

```diff
@@ -304,6 +304,22 @@ Waits for element to show up on the page.
 
 ```
 
+
+### .waitForElementToContain(selector, callback)
```

Comment:
Can you update this to match other functions `selector|element` and add test for that use case (so we won't break it in the future). Thank you.